src/Core/Domain/Manufacturer/ValueObject/ManufacturerId.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\Manufacturer\ValueObject;
+
+/**
+ * Manufacturer's identity type
+ */
+class ManufacturerId
+{
+    /**
+     * @var int
+     */
+    private $manufacturerId;
+
+    /**
+     * @param int $manufacturerId
+     */
+    public function __construct($manufacturerId)
+    {
+        $this->manufacturerId = $manufacturerId;
```

Comment:
No validation on this ID ?

Remember @eternoendless rule 😉 : if we define a DTO that behaves exactly as a primitive ... we just duplicated the wheel

Reply:
i left it on purpose and fix it while rebasing since this VO is implemented in another PR. :)